plugwise_usb/nodes/helpers/pulses.py

```
@@ -922,6 +926,10 @@ def _logs_missing(self, from_timestamp: datetime) -> list[int] | None:
             return None
 
         # We have an suspected interval, so try to calculate missing log addresses prior to first collected log
+        _LOGGER.debug(
```

Ruff (0.11.9)

Not enough arguments for `logging` format string

(PLE1206)